api/Pipfile

```diff
@@ -46,6 +46,10 @@ pytest-xdist = "==1.22.2"
 requests-mock = "==1.4.0"
 rfc3987 = "==1.3.7"
 strict-rfc3339 = "==0.7"
+pyyaml = ">=4.2b1"
```

Comment:
@andrewoh531 does it matter that the other ones are `4.2b4` but this is `4.2b1`?

Reply:
It resolves to `4.24b` in the `Pipfile.lock` but I'll update it to be consistent with the other `Pipfile`s